.github/workflows/generate-sandboxes-main.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version-file: '.nvmrc'
```

Comment:
Following the discussion and the PR that set up the `node-version-file` configuration, it was intended to not reference `.nvmrc` in these two particular cases.